README.md

```diff
@@ -43,3 +43,7 @@ $ python manage.py makemigrations
 $ python manage.py migrate
 $ python manage.py runserver
 ```
+
+* Note:
```

Comment:
May be we should not make changes in them, because they are autogenerated files. It might break.